presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -373,12 +374,24 @@ public void testExplainAnalyzeDDL()
         computeActual("EXPLAIN ANALYZE DROP TABLE orders");
     }
 
-    // Flaky test: https://github.com/prestodb/presto/issues/20764
-    @Test(expectedExceptions = RuntimeException.class, expectedExceptionsMessageRegExp = "Regexp matching interrupted", timeOut = 30_000, enabled = false)
-    public void testRunawayRegexAnalyzerTimeout()
+    @DataProvider
+    public static Object[][] timeouts()
```

Comment:
curious why you parametrized the test.

Reply:
With an optimizer timeout of 0.01s - you are much more likely to hit the other error message. But it might be overkill

Reply:
yeah, i think the point of this test is to make sure that a long running regex gets interrupted.  It's just sometimes possible that the timeout will be hit elsewhere.

Reply:
ok - I'll remove the parametrization and keep the regex